go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/getlantern/lantern-client
 
 go 1.22.0
 
-// replace github.com/getlantern/flashlight/v7 => ../flashlight
+replace github.com/getlantern/flashlight/v7 => ../flashlight
```

Comment:
I think you want to comment this out, right?

Reply:
Yep, Forgot to uncomment it. Done.